helpers/api/oracles/update-amount.ts

```diff
@@ -0,0 +1,37 @@
+import { Network_v2, fromSmartContractDecimals } from "@taikai/dappkit";
+
+import models from "db/models";
+export default async function updateAmount(events, network: Network_v2) {
```

Comment:
rename to `updateOraclesAmount` because while _it is_ on the oracles folder, while reading the code you might forget that